tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1319,310 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup [%s] with labels [%v]", scheduleBkpAllNs, labelForAllNamespace))
+			firstScheduleBackupForAllNs, err = GetFirstScheduleBackupName(ctx, scheduleBkpAllNs, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupForAllNs))
+			err = NamespaceLabelBackupSuccessCheck(firstScheduleBackupForAllNs, ctx, bkpNamespaces, labelForAllNamespace)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying if the labeled namespaces [%v] are backed up and check for labels [%s] applied to backups [%s]", bkpNamespaces, labelForAllNamespace, firstScheduleBackupForAllNs))
+			scheduleNames = append(scheduleNames, scheduleBkpAllNs)
+		})
+		Step("Restoring backup of single application", func() {
+			log.InfoD("Restoring backup of single application")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%v", manualBkpSingleNs, time.Now().Unix())
+			err = CreateRestore(restoreName, manualBkpSingleNs, nil, destinationClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup restore with name [%s] in default namespace", restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		Step("Restoring multiple applications backup", func() {
+			log.InfoD("Restoring multiple applications backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%v", manualBkpMultipleNs, time.Now().Unix())
+			err = CreateRestore(restoreName, manualBkpMultipleNs, nil, destinationClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying multiple backups [%s] restore in default namespace", restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		Step("Restoring all applications backup", func() {
+			log.InfoD("Restoring all applications backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%v", manualBkpAllNs, time.Now().Unix())
+			err = CreateRestore(restoreName, manualBkpAllNs, nil, destinationClusterName, orgID, ctx, nil)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying all backups [%s] restore in default namespace", restoreName))
+			restoreNames = append(restoreNames, restoreName)
+		})
+		Step("Restoring first scheduled backup of single namespace", func() {
+			log.InfoD("Restoring first scheduled backup of single namespace")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			restoreName = fmt.Sprintf("%s-%v", restoreNamePrefix, time.Now().Unix())
+			err = CreateRestore(restoreName, firstScheduleBackupName, nil, destinationClusterName, orgID, ctx, nil)
```

Comment:
Since for manual backups tried the default restore, try custom restores for scheduled backups

Reply:
There is another test case written for that approach